runsc/boot/config.go

```diff
@@ -227,6 +243,7 @@ func (c *Config) ToFlags() []string {
 		"--num-network-channels=" + strconv.Itoa(c.NumNetworkChannels),
 		"--rootless=" + strconv.FormatBool(c.Rootless),
 		"--alsologtostderr=" + strconv.FormatBool(c.AlsoLogToStderr),
+		"--ref-leak-mode=" + c.ReferenceLeakMode.String(),
```

Comment:
c.ReferenceLeakMode.String() will return something different than: disabled, warning, trace. You could add the reverse from `MakeRefsLeakMode()` and possibly remove `LeakMode.String()`

Reply:
Are the camel case strings used anywhere other than here? I feel like the Stringer returning "disabled", "warning" etc. is ok but it's not a big deal either way.